src/components/LoanCards/GridLoanCard.vue

```diff
@@ -196,24 +197,36 @@ export default {
 	methods: {
 		toggleFavorite() {
 			// optimistically toggle it locally first
-			this.isFavorite = !this.isFavorite;
+			// this.isFavorite = !this.isFavorite;
```

Comment:
Prob should keep this. It sets the "optimistic value" and immediately shows the new expected star state.